build/yamls/base/conf/antrea-agent.conf

```diff
@@ -34,7 +35,25 @@ featureGates:
 # Make sure it doesn't conflict with your existing interfaces.
 #hostGateway: antrea-gw0
 
-# Encapsulation mode for communication between Pods across Nodes, supported values:
+# Determines how traffic is encapsulated. It has the following options:
+# encap(default):    Inter-node Pod traffic is always encapsulated and Pod to outbound traffic is
+#                    masqueraded.
+# noEncap:           Inter-node Pod traffic is not encapsulated, but Pod to outbound traffic is
+#                    masqueraded. Underlying network must be capable of supporting Pod traffic
+#                    across IP subnets.
+# hybrid:            noEncap if worker Nodes on same subnet, otherwise encap.
+# networkPolicyOnly: Antrea enforces NetworkPolicy only, and utilizes CNI chaining and delegates Pod
+#                    IPAM and connectivity to the primary CNI.
+#
+#trafficEncapMode: encap
+
+# Whether or not to SNAT (using the Node IP) the egress traffic from a Pod to the external network.
+# This option is for the noEncap traffic mode only, and the default value is false. In the noEncap
+# mode, if the cluster's Pod CIDR is routable in the Node network, then the Pod traffic to the Node
+# or external network needs not be SNATTed.
+#noSNAT: false
```

Comment:
should this explain the relationship between `networkPolicyOnly` and `noSNAT`. It seems the intention is for SNAT to be automatically disabled for `networkPolicyOnly`, with no possibility for the user to override this